### DISCUSSION TITLE
is there any limitation of S-Group count?

### DISCUSSION THREAD

**Discussion by pa-nana** [epam/ketcher#629] [Q&A] [answered]
Hi, I followed the [instruction](https://github.com/epam/ketcher/discussions/481#discussioncomment-630626) (which is very useful) and got Ketcher embedded in our project.

- Ketcher v2.2.1 (fetch from GitHub)
- standalone

When I tried to draw something with S-Group, e.g. 796, and press the "Save As..." button, I got the message "number does not fit". In case I did something wrong in my environment, I tried on Ketcher 2.2 Standalone demo site and got the same result (see image).

I would like to know if I miss any or this is not the case supported by Ketcher 2 ? Thank you.

![image](https://user-images.githubusercontent.com/87416818/125595791-59ff88d5-e0f9-4c2e-8f61-01566158d5b0.png)

**Comment by AndreiMazol** [comment, marked as answer]
Hi @CLLPCH ,
I have analyzed the issue and the reason of it is a limitation of mol v2000 file: max number of bonds is 999 but in your case it is more than 1000 - it reproduces for single bond that is repeated 500 times). From the version 2.2 mol v3000 is supported so we need to think about suitable technical solution to solve this issue. Unfortunately it can take time so we cannot fix it right now and postpone release date of 2.3.  This fix will be delivered in scope of release 2.4. See issue https://github.com/epam/ketcher/issues/646

Thank you for rising this issue.

Best  regards,
Andrei

**Reply by pa-nana** [reply]
@AndreiMazol Thanks a lot!
